setup.py

```diff
@@ -427,6 +427,46 @@ def get_extensions():
             )
         )
 
+    # Locating video codec
+    # Should be included in CUDA_HOME for CUDA >= 10.1, which is the minimum version we have in the CI
+    video_codec_found = (
+        extension is CUDAExtension
+        and CUDA_HOME is not None
+        and os.path.exists("/usr/local/include/cuviddec.h")
+        and os.path.exists("/usr/local/include/nvcuvid.h")
+    )
```

Comment:
Can you instead use the environment variable `TORCHVISION_INCLUDE` (and `TORCHVISION_LIBRARY`) here instead?